packages/paste-website/src/pages/components/side-panel/index.mdx

```diff
@@ -0,0 +1,200 @@
+
+<StoryPreview
+  title="Basic Side Panel example"
+  storyID="components-side-panel--basic"
+  code={`
+export const SidePanelExample = (): React.ReactNode => {
+  const [isOpen, setIsOpen] = React.useState(true);
+  const sidePanelId = useUID();
+  return (
+    <SidePanelContainer id={sidePanelId} isOpen={isOpen} setIsOpen={setIsOpen}>
+      <SidePanel>
+        <SidePanelHeader>
+          <Heading as="h3" variant="heading30">Heading</Heading>
+          <SidePanelHeaderActions>
+            <Button variant="reset">
+              <MoreIcon decorative={false} title="side panel actions menu" />
+            </Button>
+          </SidePanelHeaderActions>
+        </SidePanelHeader>
+        <SidePanelBody>
+          <Separator orientation="horizontal" verticalSpacing="space0" />
+          Side Panel content goes here.
+        </SidePanelBody>
+      </SidePanel>
+      <SidePanelPushContentWrapper>
+        <Box paddingTop="space40" paddingLeft="space40" paddingRight="space40">
+          <SidePanelButton variant="secondary">Toggle Side Panel</SidePanelButton>
+        </Box>
+      </SidePanelPushContentWrapper>
+    </SidePanelContainer>
+  )`}
+/>
+
+### Internationalization
+
+To internationalize Side Panel, simply pass different text as children to the Side Panel components. The only exceptions are the close button in the SidePanelHeader and the SidePanelButton/SidePanelBadgeButton. To change the buttons' accessible label text, use the `i18nCloseSidePanelTitle` and `i18nOpenSidePanel` props on the `SidePanelContainer`.
+
+<StoryPreview
+  title="Internationalized Side Panel example"
+  storyID="components-side-panel--i-18-n"
+  code={`
+export const SidePanelExample = (): React.ReactNode => {
+  const [isOpen, setIsOpen] = React.useState(true);
+  const sidePanelId = useUID();
+  return (
+    <SidePanelContainer id={sidePanelId} isOpen={isOpen} setIsOpen={setIsOpen} i18nCloseSidePanelTitle="cerrar panel lateral" i18nOpenSidePanelTitle="abrir panel lateral">
+      <SidePanel>
+        <SidePanelHeader>
+          <Heading as="h3" variant="heading30">Título</Heading>
+        </SidePanelHeader>
+        <SidePanelBody>
+          ...
+        </SidePanelBody>
+      </SidePanel>
+      <SidePanelPushContentWrapper>
+        <Box paddingTop="space40" paddingLeft="space40" paddingRight="space40">
+          <SidePanelButton variant="secondary">Probar Panel Lateral</SidePanelButton>
+        </Box>
+      </SidePanelPushContentWrapper>
+    </SidePanelContainer>
+  )`}
+/>
+
+## Composition notes
+
+The Side Panel comes with some smaller components that can be used to compose a Side Panel to your application's needs. All of the following components should be used inside of a Side Panel Container, with Side Panel and Side Panel Push Content Wrapper being its direct children. The Side Panel Container controls the positioning of the Side Panel with relation to the page content.
```

Comment:
```suggestion
The Side Panel comes with some smaller components that can be used to compose a Side Panel to your application's needs. All of the following components should be used inside of a `SidePanelContainer`, with `SidePanel` and `SidePanelPushContentWrapper` being its direct children. The Side Panel Container controls the positioning of the Side Panel with relation to the page content.
```

Reply:
^ just bc I first missed the part that it everything had to be wrapped with `SidePanelContainer`